test-utils/src/main/java/org/hibernate/validator/testutil/ConstraintViolationAssert.java

```diff
@@ -62,8 +63,8 @@ private ConstraintViolationAssert() {
 	 * @param expectedMessages The expected constraint violation messages.
 	 */
 	public static void assertCorrectConstraintViolationMessages(Set<? extends ConstraintViolation<?>> violations,
-																String... expectedMessages) {
-		List<String> actualMessages = newArrayList();
+			String... expectedMessages) {
+		List<String> actualMessages = new ArrayList<String>();
```

Comment:
Ah, makes sense. It'd be nice to use Java 7's diamond in tests. Maven supports it, but the IDEs don't like different target levels for main and test (at least Eclipse doesn't).